src/components/InputField/component.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import {
+  Form, Input
+} from 'antd'
+import PropTypes from 'prop-types'
+
+const InputField = ({
+  placeholder, validateStatus, help, field, prefix: Prefix, form: { errors, touched }, ...inputProps
+}) => (
+  <Form.Item
+    validateStatus={(errors[field.name]) ? 'error' : null}
+    help={touched[field.name] && errors[field.name]}
```

Comment:
let's move it to the getter inside `container` now to make functional component more dumb )